<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>171</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_20">
     <item>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>167</width>
          <height>437</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_70">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>17</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(61, 234, 37);</string>
    </property>
    <property name="text">
     <string>Results:</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>521</width>
      <height>441</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_41">
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>517</width>
          <height>437</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>30</y>
      <width>201</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_15">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_27">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_57">
         <item>
          <widget class="QCheckBox" name="face_check">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>23</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(246, 25, 235)</string>
           </property>
           <property name="text">
            <string>Face</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_40">
           <property name="minimumSize">
            <size>
             <width>89</width>
             <height>23</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(246, 25, 235)</string>
           </property>
           <property name="text">
            <string>Person</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_41">
         <property name="minimumSize">
          <size>
           <width>187</width>
           <height>23</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(245, 121, 0)</string>
         </property>
         <property name="text">
          <string>Masked face</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_58">
         <item>
          <widget class="QCheckBox" name="male_check">
           <property name="minimumSize">
            <size>
             <width>90</width>
             <height>23</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(246, 25, 235)</string>
           </property>
           <property name="text">
            <string>Male</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_42">
           <property name="minimumSize">
            <size>
             <width>89</width>
             <height>23</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color:rgb(246, 25, 235)</string>
           </property>
           <property name="text">
            <string>Female</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_59">
       <item>
        <widget class="QLabel" name="label_98">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>27</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(187, 228, 51)</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="text">
          <string>Age:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_41">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_60">
         <item>
          <widget class="QLabel" name="label_99">
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-color: rgb(239, 41, 41);
background-color: rgb(114, 159, 207);
color: rgb(61, 234, 37);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="text">
            <string>From</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_42">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_24">
           <property name="minimumSize">
            <size>
             <width>27</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>10</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_61">
         <item>
          <widget class="QLabel" name="label_100">
           <property name="minimumSize">
            <size>
             <width>20</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-color: rgb(239, 41, 41);
background-color: rgb(114, 159, 207);
color: rgb(61, 234, 37);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <property name="lineWidth">
            <number>0</number>
           </property>
           <property name="text">
            <string>To </string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_43">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_25">
           <property name="minimumSize">
            <size>
             <width>27</width>
             <height>25</height>
            </size>
           </property>
           <property name="text">
            <string>40</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_111">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>190</width>
      <height>17</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 0);</string>
    </property>
    <property name="text">
     <string>                    Subjects</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_112">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>160</width>
      <height>17</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 255, 0)</string>
    </property>
    <property name="text">
     <string>                    Objetcs</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>177</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_14">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_51">
       <item>
        <widget class="QLabel" name="label_85">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(114, 159, 207)</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string>Vehicles</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_35">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="vehicles_combobox">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>select</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_52">
       <item>
        <widget class="QLabel" name="label_86">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(114, 159, 207)</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string>License plate</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_36">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>38</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_22">
         <property name="styleSheet">
          <string notr="true">border-color: rgb(204, 0, 0);</string>
         </property>
         <property name="text">
          <string>search</string>
         </property>
         <property name="placeholderText">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_53">
       <item>
        <widget class="QLabel" name="label_87">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(114, 159, 207)</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="text">
          <string>Category</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_37">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="category_combobox">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>select</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_113">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>85</width>
      <height>17</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0, 255, 0)</string>
    </property>
    <property name="text">
     <string>      Color</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>30</y>
      <width>91</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_17">
     <item>
      <spacer name="verticalSpacer_15">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>13</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_29">
       <item>
        <widget class="QLabel" name="label_104">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(173, 127, 168)</string>
         </property>
         <property name="text">
          <string>      Pick</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_105">
         <property name="styleSheet">
          <string notr="true">background-color:rgb(173, 127, 168)</string>
         </property>
         <property name="text">
          <string>     color</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_16">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>17</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="pick_color_combobox">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <item>
        <property name="text">
         <string>select</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_72">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>40</y>
      <width>227</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>227</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(44, 238, 11);
background-color: rgb(173, 127, 168);</string>
    </property>
    <property name="text">
     <string>                        Box</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_71">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>227</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>227</width>
      <height>32</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(44, 238, 11);
background-color: rgb(173, 127, 168);</string>
    </property>
    <property name="text">
     <string>                      Search</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>87</y>
      <width>229</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_42">
       <item>
        <widget class="QLabel" name="label_73">
         <property name="minimumSize">
          <size>
           <width>39</width>
           <height>29</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(114, 159, 207);
font: 11pt &quot;Ubuntu&quot;;
border-color: rgb(246, 25, 235);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
         <property name="text">
          <string>From</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit_12">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>26</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(204, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_43">
       <item>
        <widget class="QLabel" name="label_74">
         <property name="minimumSize">
          <size>
           <width>39</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgb(114, 159, 207)</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>2</number>
         </property>
         <property name="text">
          <string>To     :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit_13">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>26</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-color: rgb(204, 0, 0);</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_select_video_file">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(61, 234, 37);</string>
    </property>
    <property name="text">
     <string>Video File</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuVideo_Analytics">
    <property name="title">
     <string>Video Analytics</string>
    </property>
   </widget>
   <addaction name="menuVideo_Analytics"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
